### DISCUSSION TITLE
GItHub Education approved but not available

### DISCUSSION THREAD

**Discussion by Gamerking-MW** [community/community#132406] [GitHub Education]
### Hi everyone,

I applied for the GitHub student pack on 13.07.2024. My application was approved, but it is not yet available.

**Comment by iamdulanga** [comment]
Hello, @Gamerking-MW 

1. First congratulations for your approval.
2. Don't worry, you are successfully approved now.
3. It will take 2-4 days to complete the process.

Hope this will help.
Dulanga.

**Comment by davevad93** [comment]
Hi @Gamerking-MW ,

If the status label on your application is purple and says "`Congrats! Your request for @Gamerking-MW was approved and your benefits are now available.`", technically it means that you can access your benefits, so first try to go to the [Education learn page](https://education.github.com/learner/learn) , on the right side check if you have claimed offers:

![1](https://github.com/user-attachments/assets/510c7e2f-9357-4910-bffe-1af610226644)

Alternatively you can go [here](https://education.github.com/pack/redeem/copilot-student) to claim your free `GitHub Copilot` subscription that comes with the `Student Developer Pack`.

If you are unable to access `Copilot` there, try with try logging out and logging back in from the `Copilot` extension in your IDE e.g. `VS Code`, `Visual Studio`, `IntelliJ`, `Android Studio`, `R Studio` etcetera.

If you don't have the `Copilot` extension installed in your IDE, install it. The following steps are for `VS Code`:

1. Install the Copilot extension in `VS Code`.
2. Log in with your current GitHub account.
3. Copy the 8-digit verification code to the clipboard and paste it into the Device Activation pop-up window.
